omod/src/main/java/org/openmrs/module/fhir/providers/RestfulPatientResourceProvider.java

```diff
@@ -198,7 +198,7 @@ public MethodOutcome updatePatient(@ResourceParam Patient patient, @IdParam IdDt
 					.setDetails(
 			            "No Patient is associated with the given UUID to update. Please"
 			                    + " make sure you have set at lease one prefered identifier, non-deleted name, Gender and Birthdate to create a new "
-			                    + "Person with the given UUID. The actual error message was" + e.getMessage());
+			                    + "Patient with the given UUID. The actual error message was" + e.getMessage());
 			retVal.setOperationOutcome(outcome);
```

Comment:
Hmm.... not too convinced by the error message - surely patient update fails for reasons other than there being no patient associated with a given UUID? this may be misleading... I suggest replacing it with something like "Failed to update patient. The actual error message was  " + e.getMessage();